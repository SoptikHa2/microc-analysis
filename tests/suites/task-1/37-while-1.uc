 sum(n) {
   var x;
   x = 0;
   while (n > 0) {
     x = x + 1;
     n = n - 1;
   }

   return x;
 }

 main() {
   return sum(50000);
 }
