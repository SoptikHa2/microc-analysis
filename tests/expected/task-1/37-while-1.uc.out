---
test: run suites/task-1/37-while-1.uc
---

exit: 80
